actors/cache/impl/zcache.go

```diff
@@ -136,9 +159,18 @@ func (m *ZCache) GetRobustAddress(address address.Address) (string, error) {
 		return address.String(), nil
 	}
 
-	// This is a short address, get the robust one
-	var robustAdd string
+	// If it's a short address, try to get the associated f4
 	ctx := context.Background()
+	mapping := &EvmAddressMapping{}
+	err = m.evmAddressesMap.Get(ctx, address.String(), mapping)
+
+	if err == nil && mapping.F4 != "" {
+		// If an associated f4 is found, return this address
+		return mapping.F4, nil
+	}
+
+	// If no f4 is found, then the address is not an EVM actor type
+	var robustAdd string
```

Comment:
We have a problem with this. As the Get only tries to fetch locally, and if not found, tries remotely, whenever the running instance has a mapping with only F2 or F4 locally, It won't know if other instance fills up the remaining value.

Reply:
It could happen some indexing instances don't know about updates on the value. This is not happening with other caches, as they only write once per key